### PR TITLE
[BIA-492] Fixes postgres installation for DS 3.2.

### PR DESCRIPTION
**Pull request by DavidJGapCR**
### Testing
If you don't have a postgres database you can use this to download it:
[ed-fi - EdFi.Suite3.Ods.Populated.Template.PostgreSQL 5.2.0-b10359](https://www.myget.org/feed/ed-fi/package/nuget/EdFi.Suite3.Ods.Populated.Template.PostgreSQL)

To test this just execute the installer on a postgres database. The installer should work fine.

### CODE REVIEW

File: src/EdFi.AnalyticsMiddleTier.DataStandard32/Chrab/PostgreSQL/0003-View-ChronicAbsenteeismAttendanceFact-Update.sql [hunk 1]
```diff
@@ -3,6 +3,8 @@
 -- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 -- See the LICENSE and NOTICES files in the project root for more information.
 
+DROP VIEW IF EXISTS analytics.chrab_ChronicAbsenteeismAttendanceFact;
+
 CREATE OR REPLACE VIEW analytics.chrab_ChronicAbsenteeismAttendanceFact
```
Comment (stephenfuqua) [inline src/EdFi.AnalyticsMiddleTier.DataStandard32/Chrab/PostgreSQL/0003-View-ChronicAbsenteeismAttendanceFact-Update.sql:8]: Why would you need to add a `DROP VIEW` statement when line 6 (now 8) already has `OR REPLACE` in it?

Comment (DavidJGapCR) [reply src/EdFi.AnalyticsMiddleTier.DataStandard32/Chrab/PostgreSQL/0003-View-ChronicAbsenteeismAttendanceFact-Update.sql:8]: That surprised me too. I took this from the postgres documentation:
Capture

Here is the link: https://www.postgresql.org/docs/current/sql-createview.html

Comment (stephenfuqua) [reply src/EdFi.AnalyticsMiddleTier.DataStandard32/Chrab/PostgreSQL/0003-View-ChronicAbsenteeismAttendanceFact-Update.sql:8]: How interesting and unexpected. I guess that means the columns had changed previously. OK, thanks for the info.

**Review by kurtjohnson-HS — APPROVED** [review]
Confirmed install local. Descriptors loaded, view present and returns data. Ran a couple of times without issue -- no duped data nor DDL errors.

**Pull request merged**